app-agent/tests/relay_together.rs

```diff
@@ -15,33 +15,32 @@ mod common;
 ///
 /// Note: we use a limited set of plugins so that it works in the CI environment.
 #[test]
-fn client_to_server_to_csv_ipv4() {
-    // works in CI
-    client_to_server_to_csv_on_address(Some(("localhost", "50051"))).unwrap()
-}
+fn client_to_server_to_csv() {
+    // These tests are in the same test function because they must NOT run concurrently (same port).
```

Comment:
You should do the same thing for local.rs and relay_server.rs to fix the problems about Cargo.lock